lib/pages/child/panel_page.dart

```diff
@@ -1,27 +1,113 @@
+import 'dart:developer';
 import 'package:flutter/material.dart';
+import 'package:fokus/utils/app_locales.dart';
+import 'package:fokus/utils/theme_config.dart';
 import 'package:fokus/widgets/app_navigation_bar.dart';
 import 'package:fokus/widgets/child_wallet.dart';
+import 'package:fokus/widgets/plan_widget.dart';
 
 class ChildPanelPage extends StatefulWidget {
-	@override
-	_ChildPanelPageState createState() => new _ChildPanelPageState();
+  @override
+  _ChildPanelPageState createState() => new _ChildPanelPageState();
 }
 
 class _ChildPanelPageState extends State<ChildPanelPage> {
-	@override
-	Widget build(BuildContext context) {
-		return Scaffold(
-			body: Column(
-				crossAxisAlignment: CrossAxisAlignment.start,
-				children: [
-					ChildCustomHeader(),
-					Container(
-						padding: EdgeInsets.all(8.0),
-						child: Text('Dzisiejsze plany', textAlign: TextAlign.left, style: Theme.of(context).textTheme.headline2)
-					)
-				]
-			),
-			bottomNavigationBar: AppNavigationBar.childPage(currentIndex: 0)
-		);
-	}
+  // styling vars
+  final Color futurePlansColor = Colors.amber;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
+          ChildCustomHeader(),
+          Container(
+              child: Container(
+                  child: Flexible(
+                      child: ListView(
+            physics: BouncingScrollPhysics(),
+            children: <Widget>[
+              Padding(
+                  padding: EdgeInsets.only(
+                      bottom: AppBoxProperties.sectionPadding,
+                      right: AppBoxProperties.screenEdgePadding,
+                      left: AppBoxProperties.screenEdgePadding),
+                  child: Text(
+                      '${AppLocales.of(context).translate("page.childSection.panel.content.inProgress")} ',
+                      style: Theme.of(context).textTheme.headline1)),
```

Comment:
1. This inconsistent with what Mikołaj has (font size)
2. I already asked him to make a widget out of that so you might want to wait for his merge